share/empty.go

```diff
@@ -14,26 +14,27 @@ import (
 )
 
 var (
-	emptyRoot *Root
-	emptyEDS  *rsmt2d.ExtendedDataSquare
+	// EmptyRoot returns Root of an empty EDS.
+	EmptyRoot *Root
+	// EmptyEDS returns the EDS of the empty block data square.
+	EmptyEDS *rsmt2d.ExtendedDataSquare
```

Comment:
Previous version with funcs as getters didn't allow to modify values. Now it is possible to write to EmptyRoot, EmptyEDS vars

Reply:
Well, yeah

Reply:
Better keep it encapsulated